### PR TITLE
Enable signing using InternalKey and update version for beta package …

### PR DESCRIPTION
**Pull request by upendras**
…to 2.0.0-beta1

### CODE REVIEW

File: Signing.props [hunk 1]
```diff
@@ -4,11 +4,16 @@
...
     http://devdiv/sites/des/bars/Shared%20Documents/OneNote/VS%20Eng%20BARS%20Documentation/MicroBuild.one 
   -->
-
+  
   <PropertyGroup>
     <SignAssembly Condition="'$(SIGNING_KEYS_PATH)'!='' AND Exists('$(SIGNING_KEYS_PATH)\35MSSharedLib1024.snk')">true</SignAssembly>
   </PropertyGroup>
-
+  
+  <PropertyGroup Condition="'$(PrivateRelease)' == 'True' AND Exists('$(SIGNING_KEYS_PATH)\InternalKey.snk')">
```
Comment (SergeyKanzhelev) [inline Signing.props]: How PrivateRelease differs from public release?

File: src/Core/Managed/Core.Uwp/Core.Uwp.csproj [hunk 2]
```diff
@@ -17,10 +17,10 @@
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-    <DefineConstants>TRACE;DEBUG;UWP;UTC</DefineConstants>
+    <DefineConstants>$(DefineConstants);TRACE;DEBUG;UWP;UTC</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)' == 'Release'">
-    <DefineConstants>TRACE;CODE_ANALYSIS;CODE_ANALYSIS;CODE_ANALYSIS;UWP;UTC</DefineConstants>
+    <DefineConstants>$(DefineConstants);TRACE;CODE_ANALYSIS;CODE_ANALYSIS;CODE_ANALYSIS;UWP;UTC</DefineConstants>
```
Comment (SergeyKanzhelev) [inline src/Core/Managed/Core.Uwp/Core.Uwp.csproj]: good catch

**Comment by SergeyKanzhelev** [comment]
I don't understand why we wouldn't use publicrelease property to build interim builds. Other than that - it is good to go

File: Signing.props (hunk 1 above)
Comment (upendras) [inline Signing.props]: It signs using InternalKey.snk instead of 35MSSharedLib1024 that public releases use and is fully signed instead of delay signed.

**Pull request closed without merging**